rancher-sandbox/rancher-desktop-networking
# Add forwarding iptables rules from loopback to eth0

This allows the traffic that is destined for the loopback interface to also get picked up by the tap device `eth0`. This is to fix the scenario where the published port is bound to `127.0.0.1` e.g `docker run -d -p 127.0.0.1:1041:80 nginx`. In this scenario, there would be no accessible path to the container through `host-switch.exe` since the `vm-switch` is configured to only pick up traffic from `eth0`.

To test:
Run Rancher Desktop as non-admin or admin.
and Then:
```
docker run -d -p 127.0.0.1:1041:80 nginx
```
from windows host:
```
curl.exe localhost:1401
```
should succeed. 

Related to: https://github.com/rancher-sandbox/rancher-desktop/issues/4296
